pkg/models/repository_configuration.go

```diff
@@ -1,19 +1,19 @@
 package models
 
 import (
-	"github.com/google/uuid"
 	"github.com/lib/pq"
 	"gorm.io/gorm"
 )
 
 type RepositoryConfiguration struct {
 	Base
-	Name      string         `json:"name" gorm:"default:null"`
-	URL       string         `json:"url" gorm:"default:null"`
-	Versions  pq.StringArray `json:"version" gorm:"type:text[],default:null"`
-	Arch      string         `json:"arch" gorm:"default:null"`
-	AccountID string         `json:"account_id" gorm:"default:null"`
-	OrgID     string         `json:"org_id" gorm:"default:null"`
+	Name         string         `json:"name" gorm:"default:null"`
+	URL          string         `json:"url" gorm:"default:null"`
+	Versions     pq.StringArray `json:"version" gorm:"type:text[];default:null"`
+	Arch         string         `json:"arch" gorm:"default:null"`
+	AccountID    string         `json:"account_id" gorm:"default:null"`
+	OrgID        string         `json:"org_id" gorm:"default:null"`
+	Repositories []Repository   `json:"packages" gorm:"foreignKey:UUID"`
```

Comment:
should "json" for Repositories be "repositories"? 

Also, I think  "foreignKey" would be "ReferRepoConfig" instead of "UUID". Following the "UserRefer" example: https://gorm.io/docs/has_many.html

Reply:
thanks! it should be "repositories"

Reply:
I have just realized about a wrong understanding about GORM and relationships; thanks for this @rverdile ; I have wrong in the other relationships too. I will review and update it.